internal/transport/http2_client.go

```diff
@@ -783,6 +784,12 @@ func (t *http2Client) NewStream(ctx context.Context, callHdr *CallHdr) (*Stream,
 		h := it.(*headerFrame)
 		h.streamID = t.nextID
 		t.nextID += 2
+
+		// drain client transport if nextID > MaxStreamID, which is set to
+		// 75% of math.MaxUint32, which signals gRPC that the connection is closed
```

Comment:
Is the `75% of math.MaxUint32` important here since it is already mentioned in the comment for `MaxStreamID`?